packages/flutter/lib/src/cupertino/icons.dart

```diff
@@ -27,35 +27,64 @@ import 'package:flutter/widgets.dart';
 class CupertinoIcons {
   CupertinoIcons._();
 
+  /// The icon font used for Cupertino icons.
   static const String iconFont = 'packages/cupertino_icons/CupertinoIcons';
 
   // Manually maintained list
 
+  /// A thin left chevron.
+  /// https://github.com/flutter/cupertino_icons/blob/master/map.png
```

Comment:
I don't think linking to map.png for each of these is helpful, though maybe we could link to it from the class docs in the See also section?
```md
See also:
 * [Icon], which is used to show icons.
 * <https://github.com/flutter/cupertino_icons/blob/master/map.png>, which shows the icons in this font.
```

Reply:
Done